wpsc-components/merchant-core-v3/gateways/paypal-express-checkout.php

```diff
@@ -206,6 +349,136 @@ public function callback_ipn() {
 		exit;
 	}
 
+    /**
+	 * Pull and Record PayPal Details
+	 *
+	 * @return void
+	 */
+	public function pull_paypal_details() { 
```

Comment:
Review some of the notes here from Scrutinizer...this one method pulls down the entire codebase from 7.64 to 7.61.

https://scrutinizer-ci.com/g/wp-e-commerce/WP-e-Commerce/inspections/ff58628f-69ef-4320-9d68-fb016dc36377/code-structure/operation/%2Bglobal%5CWPSC_Payment_Gateway_Paypal_Express_Checkout%3A%3Apull_paypal_details